data.js

```diff
@@ -1,4 +1,23 @@
 module.exports = [
+  {
+    name: "Suprabha",
+    description:
+      "Hi, I'm Frontend Engineer, Content creator, Mentor, Public Speaker, Checkout my website for more details at http://suprabha.me",
+    url: "https://blog.suprabha.me/",
+    twitter: "@suprabhasupi",
+    tags: [
+      "Web Development",
+      "JavaScript",
+      "HTML",
+      "CSS",
+      "React",
+      "Vue",
+      "Front End",
+      "TypeScript",
+      "Gatsby",
+      "Web3",
+      "Webpack",
```

Comment:
Hey, @suprabhasupi. Can you, please, check the code? It looks like the `tags` array is not closed. Thanks!

Reply:
Hey @markodenic, I have resolved it. Can we merge now ☺️